docs/concepts/rules.md

```diff
@@ -121,6 +121,30 @@ CDHash   →   Binary   →   Signing ID   →   Certificate   →   Team ID
 If no rules are found that apply, scopes are then searched. See the
 [scopes.md](scopes.md) document for more information on scopes.
 
+## Rule Layering
+
+
+Since Santa is a first match system, there are some interesting ways you can
+layer rules to achieve different policies.
+
+For example if you want to allow all applications from a publisher (e.g.  North
+Pole Security) you might start with an allow rule for the Team ID to allow all
+applications from that publisher. However if you then need to prevent a specific
```

Comment:
This is a bit weird because you used North Pole Security as the example but we don't have a cloud-storage application. Could we use a different example organization?

Reply:
That's fine. I was just trying to be nice and do the gshoe thing where it's hypothetical and not calling anyone out specifically.

Reply:
ACME works